internal/grpcutil/errors.go

```diff
@@ -42,30 +39,11 @@ func ConvertGrpcToProtoError(err error) *v1alpha1.Error {
 	if err == nil {
 		return nil
 	}
-	status, ok := status.FromError(err)
-	if !ok {
-		// If the given error is not a gRPC error, return unknown
-		return &v1alpha1.Error{
-			Code:    int32(connect.CodeUnknown),
-			Message: "provided error is not a gRPC error",
-		}
+	stat, _ := status.FromError(err)
```

Comment:
Do you think it's worth it to check the `ok` return value here?  Just in case we couldn't convert this to a status? Maybe that will never happen in practice I guess.

Reply:
When `ok` is false, it still returns a status -- it just returns one with an unknown code and a message set to `err.Error()`.